cumulusci/tasks/metadata/package.py

```diff
@@ -1,12 +1,12 @@
-from builtins import str
+from builtins import str  # noqa: F401
```

Comment:
You can go ahead and remove unused imports instead of ignoring the warnings; that should have happened already.

Reply:
will do

Reply:
pushed changes